packages/components/src/primitives/hr/index.js

```diff
@@ -0,0 +1 @@
+export const Hr = 'hr';
```

Comment:
Can we name it `HorizontalRule` for those who are not familiar with this HTML shortcut?